packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-blocks-manage-button.js

```diff
@@ -34,11 +34,16 @@ function ReusableBlocksManageButton( { clientId } ) {
 						'blocks',
 						reusableBlock.attributes.ref
 					),
+				reusableBlockId: reusableBlock.attributes.ref,
```

Comment:
Instead of using the `useEntityBlockEditor` hook below, you can use the `getBlockCount` selector from the editor store.

```suggestion
				innerBlockCount: getBlockCount( clientId ),
```